scripts/data/src/org/akvo/gae/remoteapi/ExtractImageGeotag.java

```diff
@@ -0,0 +1,292 @@
+/*
+ *  Copyright (C) 2019 Stichting Akvo (Akvo Foundation)
+ *
+ *  This file is part of Akvo FLOW.
+ *
+ *  Akvo FLOW is free software: you can redistribute it and modify it under the terms of
+ *  the GNU Affero General Public License (AGPL) as published by the Free Software Foundation,
+ *  either version 3 of the License or any later version.
+ *
+ *  Akvo FLOW is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU Affero General Public License included below for more details.
+ *
+ *  The full license text can also be seen at <http://www.gnu.org/licenses/agpl.html>.
+ */
+
+package org.akvo.gae.remoteapi;
+
+import static org.akvo.gae.remoteapi.DataUtils.batchSaveEntities;
+
+import java.io.BufferedInputStream;
+import java.io.OutputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.File;
+import java.net.URLConnection;
+import java.nio.file.Paths;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+
+import org.akvo.flow.util.FlowJsonObjectReader;
+import org.apache.commons.io.IOUtils;
+import org.waterforpeople.mapping.domain.SurveyInstance;
+import org.waterforpeople.mapping.domain.response.value.Location;
+import org.waterforpeople.mapping.domain.response.value.Media;
+import org.waterforpeople.mapping.serialization.response.MediaResponse;
+
+//import com.fasterxml.jackson.core.type.TypeReference;
```

Comment:
Why can't we remove it instead of commenting out?

Reply:
We can.